pkg/rancher-desktop/main/mainEvents.ts

```diff
@@ -112,6 +112,14 @@ interface MainEventNames {
    * quitting.
    */
   'quit'(): void;
+
+  /**
+   * Emitted when the "locked" state is requested. This state is enabled when
+   * a snapshot is being created or restored. It disables the UI, prevents the
+   * user from making changes to settings, and possibly prevents other actions
+   * that could mess with snapshot operations.
```

Comment:
That means that we don't get an event on unlock? (Nobody actually handles this, so it's fine for now, it's just… oddly asymmetric.)

Consider adding a line along the lines of _If the passed-in string is empty, an unlock is requested_.

Reply:
Sorry, that was confusing. I rewrote the comment - hopefully it's better now.